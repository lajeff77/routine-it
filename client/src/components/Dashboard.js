import React from 'react';
import DashboardHeader from './DashboardHeader';

const Dashboard = () => {
    return (
        <DashboardHeader />
    )
}

export default Dashboard;