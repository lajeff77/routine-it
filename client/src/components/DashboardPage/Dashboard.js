import React from 'react';
import DashboardHeader from './DashboardHeader';
import RoutineList from './RoutineList';

const Dashboard = (props) => {
    return (
        <div>
            <DashboardHeader />
            <RoutineList routineList={props.routineList} />
        </div>
    )
}

export default Dashboard;