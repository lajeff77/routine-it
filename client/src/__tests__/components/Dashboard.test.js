import { render } from "@testing-library/react";
import {expect, test} from '@jest/globals';
import React from "react";
import Dashboard from "../../components/Dashboard";


describe(Dashboard.name, ()=>{
    test('Dashboard matches snapshot', () => {
        const {container} = render(<Dashboard/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});