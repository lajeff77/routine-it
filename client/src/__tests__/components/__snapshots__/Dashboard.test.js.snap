// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Dashboard matches snapshot 1`] = `<DocumentFragment />`;
